src/html5/parser.rs

```diff
@@ -3878,8 +3943,9 @@ mod test {
 
     #[test]
     fn is_in_scope_7() {
+        let document = Document::shared();
         let mut stream = InputStream::new();
-        let mut parser = Html5Parser::new(&mut stream);
+        let mut parser = Html5Parser::new(&mut stream, document.clone());
```

Comment:
```suggestion
        let mut parser = Html5Parser::new(&mut stream, Document::clone(&document));
```

This usage makes people think of `Rc::clone(&reference)`, and communicates to the developer that it's a shallow copy of a reference rather than a deep copy of the contents.

Reply:
Etc.